contracts/coordinator/src/msg.rs

```diff
@@ -18,11 +18,21 @@ pub enum ExecuteMsg {
     SetActiveVerifiers {
         next_worker_set: WorkerSet,
     },
+    SetNextVerifiers {
+        next_worker_set: WorkerSet,
+    },
+    AddSupportedChainsForWorker {
+        chains: Vec<ChainName>,
+        worker: Addr,
+    },
```

Comment:
Why do we need this method? Can't this info just be inferred from which provers are calling `SetActiveVerifiers` and `SetNextVerifiers`?